salt/modules/aptpkg.py

```diff
@@ -2464,7 +2464,7 @@ def owner(*paths):
     return ret
 
 
-def info_installed(*names):
+def info_installed(*names, failhard=True):
```

Comment:
This form is valid in Python 3 but not in Python2, where it is a syntax error. IIRC, in Py2 you need to switch the order of these.